edgelet/workload/src/models/trust_bundle_response.rs

```diff
@@ -1,39 +1,46 @@
-/*
+/* 
  * IoT Edge Module Workload API
  *
  * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
  *
- * OpenAPI spec version: 2018-06-28
- *
+ * OpenAPI spec version: 2020-07-22
```

Comment:
I think version for existing models should not be changed since there was no change to trust bundle response. Same applies to other classes

Reply:
Understood. This was part of the generated code.